src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -1083,58 +1032,42 @@ private static TextEdit BuildRevertedEdit(TextEdit provisionalTextEdit)
-                return default;
+                return null;
             }
 
-            var onAutoInsertParams = new VSInternalDocumentOnAutoInsertParams
+            var hoverParams = new TextDocumentPositionParams()
             {
                 TextDocument = new TextDocumentIdentifier()
                 {
                     Uri = delegationDetails.Value.ProjectedUri,
                 },
                 Position = request.ProjectedPosition,
-                Character = request.Character,
-                Options = request.Options
             };
 
-            var response = await _requestInvoker.ReinvokeRequestOnServerAsync<VSInternalDocumentOnAutoInsertParams, VSInternalDocumentOnAutoInsertResponseItem?>(
-               delegationDetails.Value.TextBuffer,
-               VSInternalMethods.OnAutoInsertName,
-               delegationDetails.Value.LanguageServerName,
-               onAutoInsertParams,
-               cancellationToken).ConfigureAwait(false);
+            var response = await _requestInvoker.ReinvokeRequestOnServerAsync<TextDocumentPositionParams, VSInternalHover?>(
+                delegationDetails.Value.TextBuffer,
+                Methods.TextDocumentHoverName,
+                delegationDetails.Value.LanguageServerName,
+                hoverParams,
+                cancellationToken).ConfigureAwait(false);
+
             return response?.Response;
         }
 
-        public override Task<VSInternalHover?> HoverAsync(DelegatedPositionParams request, CancellationToken cancellationToken)
-            => DelegateTextDocumentPositionRequestAsync<VSInternalHover>(request, Methods.TextDocumentHoverName, cancellationToken);
-
-        public override Task<Location[]?> DefinitionAsync(DelegatedPositionParams request, CancellationToken cancellationToken)
-            => DelegateTextDocumentPositionRequestAsync<Location[]>(request, Methods.TextDocumentDefinitionName, cancellationToken);
-
-        public override Task<DocumentHighlight[]?> DocumentHighlightAsync(DelegatedPositionParams request, CancellationToken cancellationToken)
-            => DelegateTextDocumentPositionRequestAsync<DocumentHighlight[]>(request, Methods.TextDocumentDocumentHighlightName, cancellationToken);
-
-        public override Task<SignatureHelp?> SignatureHelpAsync(DelegatedPositionParams request, CancellationToken cancellationToken)
-            => DelegateTextDocumentPositionRequestAsync<SignatureHelp>(request, Methods.TextDocumentSignatureHelpName, cancellationToken);
-
-        public override Task<ImplementationResult> ImplementationAsync(DelegatedPositionParams request, CancellationToken cancellationToken)
-            => DelegateTextDocumentPositionRequestAsync<ImplementationResult>(request, Methods.TextDocumentImplementationName, cancellationToken);
-
-        private async Task<TResult?> DelegateTextDocumentPositionRequestAsync<TResult>(DelegatedPositionParams request, string methodName, CancellationToken cancellationToken)
```

Comment:
Looks like a lot of recent changes that have been reverted in this file, including support for Go to Implementation entirely :o

Gonna skip the rest of this file for now.